django_stomp/services/producer.py

```diff
@@ -20,9 +21,25 @@
 
 
 class Publisher:
-    # headers that must be purged from messages if present
-    # 'message-id': RabbitMQ uses it internally
-    UNSAFE_OR_RESERVED_BROKER_HEADERS_FOR_REMOVAL = ["message-id"]
+    """
+    Class used to publish messages to ActiveMQ/RabbitMQ brokers using STOMP protocol. Some headers are removed
```

Comment:
**thought:** Theoretically, it can be used with any broker that uses the STOMP protocol. But, nothing guaranteed!